### ISSUE TITLE
What to submit for problem sets

### ISSUE THREAD

**Issue by rickecon** [UC-MACSS/persp-analysis#45]
As described in the [syllabus for this class](https://github.com/UC-MACSS/persp-analysis/blob/master/README.md), you will submit 4 problem sets during the last half of this term. These problem sets will primarily involve writing code. I want you to submit your assignments in a particular way.

1. Your assignment submission will involve two parts: (a) your Python code, and (b) a PDF document that you compile using LaTeX that has your answers.
2. Your Python code should use Python 3.5 or higher. I recommend that you download the [Anaconda distribution](https://www.continuum.io/downloads) of Python from Continuum Analytics.
3. In your code, you should label your sections with the particular part of the Problem Set that that section of code is solving. Below is an example.
```python
'''
------------------------------------------------------------------------
Exercise 1a: Simulate the data
------------------------------------------------------------------------
plot_1a     = Boolean, =True if make a plot of one series of the
              simulated income data
norm_errors = (lf_years, num_draws) matrix, normally distributed errors
              with mean 0 and standard deviation sigma
------------------------------------------------------------------------
'''
plot_1a = True
norm_errors = np.random.normal(0, sigma, (lf_years, num_draws))
```
4. You should use a `print` command for any answers that your code produces in response to things the Problem Set asks for. In this way, we can just run your script and see what answers it produces.
```python
print('1b. Percent of students getting more than $100k in first period: ',
      inc0_gt100k_pct * 100, '%')
```
The code above produces the following output when I run this script.
```
1b. Percent of students getting more than $100k in first period: x.xx %
```
5. With regard to plots that the Problem Set asks for, follow the template in Issue #47 about having your script save those plots to an images folder. Further, make sure that your plot is included in the PDF document that you submit.
6. For your PDF document with your answers that you produce using LaTeX, I have included a [LaTeX tutorial](https://github.com/UC-MACSS/persp-analysis/blob/master/handouts/LaTeX/LaTeX_tutorial.pdf) document as well as a [LaTeX problem set template](https://github.com/UC-MACSS/persp-analysis/blob/master/handouts/LaTeX/LaTeX_probset_template.tex) in this repository. Note that when using the LaTeX problem set template, you must have the image document (pencildrawing.png) in the same folder as the template in order to compile the PDF. If you are not using any images, you can just delete or comment out that section of the template.

In summary, we will grade your assignments based on your code that you submit, what output it gives when we execute it, and your accompanying PDF with your answers.
